src/renderers/noop/ReactNoop.js

```diff
@@ -136,10 +136,22 @@ var NoopRenderer = ReactFiberReconciler({
   },
 
   scheduleAnimationCallback(callback) {
+    if (scheduledAnimationCallback) {
+      throw new Error(
+        'Scheduling an animation callback twice is excessive. ' +
+        'Instead, keep track of whether the callback has already been scheduled.'
+      );
```

Comment:
What does this error protect against? Seems unnecessarily dangerous. If the scheduler calls `scheduleAnimationCallback` and a callback is already scheduled, the renderer can just treat it as a noop.